### PR TITLE
[onert] Frontend for ArgMinMax

### PR DESCRIPTION
**Pull request by hseok-oh**
- Support ArgMin on loader and nnapi frontend
- Set is_arg_max param
- Remove operand type duplication in loader: handle on operation validator

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Related issue: #5115
Part of #5132

### CODE REVIEW

File: runtime/onert/frontend/base_loader/include/base_loader.h [hunk 1]
```diff
@@ -1235,21 +1236,21 @@ void BaseLoader<LoaderDomain>::loadArgMax(const Operator *op, ir::Graph &subg)
...
 
-  auto &axisOperand = subg.operands().at(am->getInputs().at(ir::operation::ArgMinMax::Input::AXIS));
-  if (!(axisOperand.operandSize() == 4 && (axisOperand.typeInfo().type() == ir::DataType::INT32 ||
-                                           axisOperand.typeInfo().type() == ir::DataType::INT64)))
-    throw std::runtime_error("ArgMax: `axis` with an int32 or int64 element is only supported.");
+  loadOperationTo<ir::operation::ArgMinMax>(op, subg, param);
+}
+
+template <typename LoaderDomain>
+void BaseLoader<LoaderDomain>::loadArgMin(const Operator *op, ir::Graph &subg)
```
Comment (glistening) [inline runtime/onert/frontend/base_loader/include/base_loader.h:1246]: @hseok-oh What about introducing `loadArgMinMax(`const Operator *op, ir::Graph &subg, bool is_argmin)`?

File: runtime/onert/frontend/base_loader/include/base_loader.h [hunk 2]
```diff
@@ -1231,25 +1231,16 @@ void BaseLoader<LoaderDomain>::loadWhile(const Operator *op, ir::Graph &subg)
...
-      break;
-    default:
-      throw std::runtime_error("ArgMax: `output_type` must be either int32 or int64.");
-  }
-  auto am = loadOperationTo<ir::operation::ArgMinMax>(op, subg, param);
+  const auto output_type = is_argmax ? op->builtin_options_as_ArgMaxOptions()->output_type()
+                                     : op->builtin_options_as_ArgMinOptions()->output_type();
+  param.output_type = tensorTypeToDataType(output_type);
+  param.is_arg_max = is_argmax;
+  ;
```
Comment (ragmani) [inline runtime/onert/frontend/base_loader/include/base_loader.h:1241]: Could you remove this?

**Review by glistening — APPROVED** [review]
LGTM

File: runtime/onert/frontend/base_loader/include/base_loader.h (hunk 2 above)
Comment (hseok-oh) [reply runtime/onert/frontend/base_loader/include/base_loader.h]: I've updated. Thanks.

**Review by ragmani — APPROVED** [review]
LGTM

**Pull request merged**